test/alternator/test_returnconsumedcapacity.py

```diff
@@ -291,3 +291,49 @@ def test_long_update(test_table):
         ExpressionAttributeValues={':val1': val1},
         ReturnConsumedCapacity='TOTAL', ReturnValues='ALL_OLD')
     assert 3 == response['ConsumedCapacity']["CapacityUnits"]
+
+# Validate that when getting a batch of requests
+# The RCU returned is calculated as each of the requests
+# separately
+def test_simple_batch_get_items(test_table_sb):
+    p1 = random_string()
+    val = random_string()
+    c1 = random_bytes()
+    test_table_sb.put_item(Item={'p': p1, 'c': c1, 'att': val}, ReturnConsumedCapacity='TOTAL')
+
+    p2 = random_string()
+    c2 = random_bytes()
+    test_table_sb.put_item(Item={'p': p2, 'c': c2, 'att': val}, ReturnConsumedCapacity='TOTAL')
+
+    response = test_table_sb.meta.client.batch_get_item(RequestItems = {
+            test_table_sb.name: {'Keys': [{'p': p1, 'c': c1}, {'p': p2, 'c': c2}], 'ConsistentRead': True}}, ReturnConsumedCapacity='TOTAL')
+    assert 'ConsumedCapacity' in response
+    assert 'TableName' in response['ConsumedCapacity'][0]
+    assert response['ConsumedCapacity'][0]['TableName'] == test_table_sb.name
+    assert 2 == response['ConsumedCapacity'][0]["CapacityUnits"]
+
+# Validate that when getting a batch of requests
+# From multiple tables we get an RCU for each of the tables
+# We also validate that the eventual consistency return half the units
+def test_multi_table_batch_get_items(test_table_s, test_table):
+    keys1 = []
+    for i in range(5):
+        p = random_string()
+        test_table_s.put_item(Item={'p': p})
+        keys1.append({'p': p})
+    keys2 = []
+
```

Comment:
nitpick: the blank line here is strange, wouldn't it better to move it before the "response = " line?

Reply:
done